### ISSUE TITLE
Focus state not visible in Windows High Contrast theme

### ISSUE THREAD

**Issue by joe-watkins** [18F/web-design-standards#2147]
## Description

I noticed the focus state, when defined as `box-shadow` does not work in Windows High Contrast theme.

## Steps to reproduce the issue

1. Visit https://standards.usa.gov/components/buttons/ in and activate Windows High Contrast theme.
2. Observe focus state is missing.

## Additional information [optional]

* `box-shadow` does not hold up in Windows High Contrast theme.
* Perhaps a media query detecting High Contrast and deliver a standard outline/border.

![2017-09-19_15-30-22](https://user-images.githubusercontent.com/3695795/30618803-e94e7f3a-9d50-11e7-844d-034499515010.png)

Keep up the awesome work!!

**Comment by msecret** [comment]
Hi @joe-watkins thanks for looking into this and reporting it to us! We did make a change to the focus state in the last sprint iteration, but haven't released that change yet. It could fix this problem though: https://github.com/18F/web-design-standards/pull/2112.

**Comment by maya** [comment]
Hi @joe-watkins, there were some updates made to the focus state (using outline instead of box-shadow) and wanted to check if this is still an issue for you?

**Comment by joe-watkins** [comment]
@maya @msecret Nailed it! ;) Good work.

![Screenshot of high contrast mode showing focus state](https://user-images.githubusercontent.com/3695795/31521718-6f5680c6-af5f-11e7-902e-973e7eff2e95.gif)

**Comment by maya** [comment]
Glad to hear! Thank you 🎉
